packages/swap/src/routes/__tests__/v2/quote.test.ts

```diff
@@ -612,7 +612,357 @@ describe('server', () => {
         'cf_swap_rate_v2',
         { asset: 'ETH', chain: 'Ethereum' },
         { asset: 'USDC', chain: 'Ethereum' },
-        '0x3782dace9d9493e', // 2.5e17 + 3 * 6250 (ingressFee)
+        '0x3782dace9d9493e', // 2.5e17 + 3/4 * 25000 (ingressFee surcharge)
+        [],
+      );
+    });
+
+    it('gets the DCA quote with a boost quote', async () => {
```

Comment:
i figured that we dont test the calculations for a boosted dca quote at the moment while working on https://github.com/chainflip-io/chainflip-sdk-monorepo/pull/1038